### PR TITLE
#27 implement social login api

### PR DESCRIPTION
**Pull request by Mir00r**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -263,6 +263,57 @@
             <scope>runtime</scope>
         </dependency>
 
+        <!--SOCIAL DEPENDENCIES START-->
```
Comment (sayemkcn) [inline pom.xml:266]: Holy shit, lot of dependencies!!
I got afraid. Will see later when I get some time for coffee.
Thanks for the contribution 😄

**Comment by vik-y** [comment]
+1 For this PR, nice feature to have.

**Comment by sayemkcn** [comment]
@Mir00r, You should consider this PR seriously.

**Comment by sayemkcn** [comment]
This pull request seems incompatible and hasn't been updated for a long time. Closing it.

**Pull request closed without merging**